integration-tests/tests/hacbs/add-component-to-application-via-hacbs-tab.spec.ts

```diff
@@ -45,44 +47,90 @@ describe('Create Components using the HACBS UI', () => {
-            cy.get(actionsDropdown.dropdown).click();
-            cy.contains('Add component').click();
+            HACBSApplications.clickActionsDropdown('Add component');
         });
 
         it('Add a component to Application', () => {
-            HACBSApplications.createComponent(publicRepos[2], componentNames[3]);
+            HACBSApplications.createComponent(publicRepos[2], componentNames[3], integrationTestNames[3]);
             HACBSApplications.createdComponentExists(componentNames[3], applicationName);
         });
     });
+
+    describe('Explore Integration Tests Tab', () => {
+        it("Click on 'Integration tests' tab and check the List View", () => {
+            HACBSApplications.goToIntegrationTestsTab();
+
+            for (let i = 0; i < 4; i++) {
+                integrationTestsTabPage.checkRowValues(integrationTestNames[i], containerImage, 'Mandatory', pipelineName);
+            }
+        });
+
+        it("Add a new Integration Test using 'Actions' dropdown, and mark it as Optional for release", () => {
+            HACBSApplications.clickActionsDropdown('Add integration test');
+            addIntegrationTestStep(integrationTestNames[4], true);
+
+            integrationTestsTabPage.checkRowValues(integrationTestNames[4], containerImage, 'Optional', pipelineName);
+        });
+
+        it("Filter Integration tests by 'Name'", () => {
+            // No Integration test should be visible
+            integrationTestsTabPage.filterByName('nothing');
+            cy.contains('No results found');
+
+            integrationTestsTabPage.filterByName('my-test');
+            // Only the first 4 Integration tests with prefix "my-test" should be visible
+            for (let i = 0; i < 4; i++) {
+                integrationTestsTabPage.checkRowValues(integrationTestNames[i], containerImage, 'Mandatory', pipelineName);
+            }
+
+            // Only the 1 Integration test should be visible, which is Optional for release
+            integrationTestsTabPage.filterByName('optional');
+            integrationTestsTabPage.checkRowValues(integrationTestNames[4], containerImage, 'Optional', pipelineName);
+        });
+
+        it("Explore the Integration Test Details page (with the Optional tag) and delete it", () => {
+            cy.contains(integrationTestNames[4]).click();
+            integrationTestsTabPage.checkMetadata(integrationTestMetadata);
+
+            HACBSApplications.clickActionsDropdown('Delete');
+            cy.get(actions.deleteModalButton).click();
+            cy.get(integrationTestNames[4]).should('not.exist');
+        });
+
+        it("Delete all the remaining Integration Tests from the list view", () => {
+            for (let i = 0; i < 4; i++) {
+                integrationTestsTabPage.deleteIntegrationTest(integrationTestNames[i]);
+                cy.get(integrationTestNames[i]).should('not.exist');
+            }
+        });
```

Comment:
Okay sure. My aim is to iterate over all the elements in the `integrationTestNames` array except the last element (because it is marked "Optional" for release). So, I'll just use something like:
`for (let i = 0; i < integrationTestNames.length-1; i++)`